### DISCUSSION TITLE
How to configure storybook with vue and react at same time

### DISCUSSION THREAD

**Discussion by Leo-July** [storybookjs/storybook#24693] [Help] [answered]
I want storybook to support vue and react at same time.How to configure the storybook. 
Tanks

**Comment by woodsboe** [comment, marked as answer]
Hi @Leo-July,
I don't think you can do it in a single Storybook instance.

Try looking at a monorepo approach. Here's a [link](https://blog.munafio.com/building-a-monorepo-using-turborepo-with-react-js-and-vue-js-b6b25d7e41f7) to an article that might inspire you.

In a monorepo, you can have multiple Storybook instances running simultaneously, and with Storybook Composition, you can compose them into a main Storybook site.

This [article](https://medium.com/@Seb_L/compose-your-turborepos-storybooks-and-deploy-them-to-vercel-94befbb78a56) will walk you through setting up a Turborepo with Storybook.
